common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/WorksIndexConfig.scala

```diff
@@ -211,18 +211,18 @@ object SourceWorkIndexConfig extends WorksIndexConfig {
 
 object MergedWorkIndexConfig extends WorksIndexConfig {
 
-  val fields = Seq.empty
-  val dynamicMapping = DynamicMapping.False
-}
-
-object IdentifiedWorkIndexConfig extends WorksIndexConfig {
-
   val fields = Seq(
     keywordField("type"),
     objectField("data").fields(
       collectionPath(analyzedPath)
     )
   )
+  val dynamicMapping = DynamicMapping.False
```

Comment:
Might have missed the conversation - but why were we changing this?

Reply:
It has been `false` for the pipeline storage indices for a while, although I think we should change it for the other index too (was about to set up a discussion for this).

Reply:
The main reason is that we then cannot query on certain fields, and we have more than just the API that uses the index (tools / reporting etc etc).